ntbs-service/appsettings.json

```diff
@@ -11,5 +11,10 @@
   "AppConfig" : {
     "AuditingEnabled" : true,
     "LegacySearchEnabled" : true
+  },
+  "AdConnectionSettings": {
+    "DomainName": "adds-ntbs.uksouth.cloudapp.azure.com",
+    "UserName": "vmAdmin",
+    "Password": ""
```

Comment:
For local testing, should get this password/use your own credentials. Still todo: Add this to yml config and secrets. We could also create a dedicated user, although vmAdmin seemed suitable

Reply:
I feel like we should create a dedicated user with the correct rights to read what it needs to read but perhaps not full vm admin rights..